### DISCUSSION TITLE
Ability to add multiple Accounts (AWS, GCP and Azure) of the same provider

### DISCUSSION THREAD

**Discussion by mantewari** [cloud-carbon-footprint/cloud-carbon-footprint#1150] [Ideas]
Hi Team,

Having the ability to add multiple accounts for the same cloud provider ( even if they are not under the same billing account). Once we do a guided install it removes the previously configured Account (of the same provider). It will be great to be able to add multiple accounts of the same provider.

**Comment by ericksod** [comment]
Hi @mantewari 
This is something that has come up a few times now. It's something that would likely be a pretty major change to implement.
Had you thought of any ways you might go about it within CCF or just suggesting the idea?
We may not currently have the bandwidth to do this at the moment, but are happy to support it if anyone wants to contribute it.
